rosdep/python.yaml

```
@@ -10069,6 +10069,10 @@ python3-tabulate:
   nixos: [python3Packages.tabulate]
   rhel: [python3-tabulate]
   ubuntu: [python3-tabulate]
+python3-tactigon-gear-pip:
+  ubuntu:
```

We can use a [wildcard OS name](https://ros.org/reps/rep-0111.html#wildcard-os-name) here so that the rule applies to pip on all platforms.

```suggestion
  '*':
```